python/foxglove-sdk-examples/logging-context/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "logging-context"
+version = "0.0.0"
+description = "Example of using Context"
+authors = [{ name = "Foxglove", email = "support@foxglove.dev" }]
+readme = "README.md"
+requires-python = ">=3.9"
+# dependencies = ["foxglove-sdk>=0.8.0"]
```

Comment:
This references a local install below, until the next version is published. See FG-11340.